src/docfx/template/liquid/LiquidTemplate.cs

```diff
@@ -122,7 +139,18 @@ public Template GetTemplate(DotLiquid.Context context, string templateName)
             {
                 return _templates.GetOrAdd(
                     templateName,
-                    new Lazy<Template>(() => LoadTemplate(Path.Combine(_templateDir, "_includes", templateName + ".liquid")))).Value;
+                    new Lazy<Template>(() =>
+                        {
+                            try
+                            {
+                                return LoadTemplate(Path.Combine(_templateDir, "_includes", templateName + ".liquid"));
+                            }
```

Comment:
add a test for this when inclusion .liquid not found?

Reply:
As I can see, this method is not in use yet

Reply:
Seems like it is used for inclusion inside the liquid template. Since it is not related to this fix, leave it for now.